doc/whats_new.rst

```diff
@@ -31,6 +31,8 @@ Bug
 
 - Fix bug in :func:`mne.inverse_sparse.mixed_norm` and :func:`mne.inverse_sparse.tf_mixed_norm` where ``weights`` was supplied but ``weights_min`` was not, by `Eric Larson`_
 
+- Fix bug in :func:`mne.io.Raw.plot` when using HiDPI displays and the MacOSX backend of matplotlib by `Eric Larson`_
```

Comment:
Maybe mention that the bug occurred when resizing? For whatever reason, the initial layout was correct.

Reply:
For me it was incorrect to start, so might as well leave the more general (current) version